utils/interop-ribs/lint-baseline.xml

```diff
@@ -1,4 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<issues format="6" by="lint 7.1.3" type="baseline" client="gradle" dependencies="false" name="AGP (7.1.3)" variant="all" version="7.1.3">
+<issues format="6" by="lint 7.4.1" type="baseline" client="gradle" dependencies="false" name="AGP (7.4.1)" variant="all" version="7.4.1">
+
+    <issue
+        id="NewApi"
```

Comment:
I had this issue before, it went away with agp update

Reply:
Which AGP version should remove this false positive?

Reply:
I think it wasn't a specific version but rather combination kotlin + agp